libcxxabi/include/__cxxabi_config.h

```diff
@@ -109,4 +110,38 @@
 #  define _LIBCXXABI_NOEXCEPT noexcept
 #endif
 
+#if __has_include(<ptrauth.h>)
+#  include <ptrauth.h>
+#endif
+
+#if defined(__APPLE__) && __has_feature(ptrauth_qualifier)
```

Comment:
It looks like that `__has_feature(ptrauth_qualifier)` should be replaced with `__has_extension(ptrauth_qualifier)`. It is defined as `EXTENSION` in clang/include/clang/Basic/Features.def, and existing tests use `__has_extension` instead of `__has_feature` for this.